scripts/modules.js

```diff
@@ -85,3 +88,60 @@ export const modulesList = async ({ search, visibility = "", page = 1 }) => {
     loadingSpinner.stop();
   }
 };
+
+export const modulesGet = async (id) => {
+  const loadingSpinner = ora("Loading Module").start();
+
+  const [moduleResponse, appListResponse] = await Promise.all([
+    apiClient.get({
+      path: `/v1/catalog/module/${id}`
+    }),
+    apiClient.get({
+      path: "/v2/apps",
+      params: {
+        limit: 1
+      }
+    })
+  ]);
+
+  loadingSpinner.stop();
+
+  if (!moduleResponse.ok) {
+    if (moduleResponse.status === 404) {
+      invalid(`Cannot find requested module with id ${id}.`);
+    } else {
+      invalid("Unable to get module. Please login and try again.");
```

Comment:
Nit: Should we maybe limit the login part of the message for only it's specific to being unauthorized? 

A case that can easily happen is a 403, for example. Or if there's an issue with our servers. In these cases, I think we should just tell the user to try again, but not necessarily attempt to login again